kod/object/passive/spell/animate.kod

```diff
@@ -204,6 +206,24 @@ messages:
       return;      
    }
 
+   EndEnchantment(who=$, report=TRUE)
+   {
+      local oMaster;
+
+      oMaster = Send(who, @GetMaster);
+      if report AND IsClass(oMaster, &User)
+      {
+         Send(oMaster, @MsgSendUser, #message_rsc=Animate_expire_rsc , #parm1=Send(who, @GetDef), #parm2=Send(who, @GetName));
```

Comment:
Remove extra space.

Reply:
Done.  Not sure how that even got there.